arch/riscv/dts/Makefile

```diff
@@ -2,6 +2,7 @@
 
 dtb-$(CONFIG_TARGET_AX25_AE350) += ae350.dtb
 dtb-$(CONFIG_TARGET_HIFIVE_U540) += hifive_u540.dtb
+dtb-$(CONFIG_NVDLA_IOFPGA) += hifive_u540_nvdla_iofpga.dtb
```

Comment:
For upstream U-boot, we'll want something like CONFIG_TARGET_HIFIVE_U540_NVDLA, but that's a little long and obnoxious. This is good enough for this branch.